soccer/motion/TrapezoidalMotionTest.cpp

```diff
@@ -3,17 +3,24 @@
 #include <Geometry2d/Point.hpp>
 #include <motion/TrapezoidalMotion.hpp>
 #include <math.h>
+#include <iostream>
 
+using namespace std;
 
 bool trapezoid1(float t, float &posOut, float &speedOut) {
```

Comment:
This isn't a seperate test case. Its just a hlper function for a couple other test cases. I put in the expect in here for a santiy check of consistency among my functions. It works fine (if it doesn't exactly give the best error messages).